### ISSUE TITLE
Can't set offset to 0 after changing it to something else

### ISSUE THREAD

**Issue by Nadwey** [GDColon/URLX#13]
That's probably because 0 == false in js.
[Editor.js:253](https://github.com/GDColon/URLX/blob/main/classes/Editor.js#L253)

This also happens to BPM, Divide and Volume.
![offset](https://user-images.githubusercontent.com/81181783/168902043-3f09713d-ff60-4430-9050-0f7dd125c0e4.gif)

**Comment by GDColon** [comment]
Yep that's a bug, but intentional for BPM and Division since they can't be zero
